src/core/cpu_disasm.cpp

```diff
@@ -434,8 +434,46 @@ void CPU::FormatComment(SmallStringBase* dest, const Instruction inst, u32 pc, c
     else if (std::strncmp(str, "offsetrs", 8) == 0)
     {
       const s32 offset = static_cast<s32>(inst.i.imm_sext32());
-      dest->append_format("{}addr={:08X}", dest->empty() ? "" : ", ",
-                          regs->r[static_cast<u8>(inst.i.rs.GetValue())] + offset);
+      u32 tempValueWord;
+      u16 tempValueHalfWord;
+      u8 tempValueByte;
+      bool returnValue=false;
+      u32 tempAddress = (regs->r[static_cast<u8>(inst.i.rs.GetValue())] + offset);
+
+      SmallString instructionStr;
+      CPU::DisassembleInstruction(&instructionStr, pc, inst.bits);
+
+      if (instructionStr[1] == 'b')
```

Comment:
Rather than disassembling the instruction (slow), you can look at the instruction's `op`, and compare with `op == InstructionOp::lb || op == InstructionOp::sb`.

Reply:
Thanks, that is indeed a lot easier. Didn't realise that InstructionOp:: existed. 

I will probably not include store opcode checks as the existing contents only matter during loads.

I'll clean it up (can save a few lines of code with that knowledge) and resubmit.